rubyric/example.rst

```diff
@@ -1,8 +1,21 @@
 Example Rubyric exercise
 ========================
 
+The following examples will show you how Rubyric exercises work. Please submit
+your exercise and contact us through email aplusguru@cs.aalto.fi. We will show
```

Comment:
Do people from <add country here> send emails to aplusguru?

Reply:
??

Reply:
Like, all the people all around the world, or just people from aalto/cs dept?